File: app/src/main/java/com/jermaine/awesomelibraries/view/activity/viewitem/ViewItemContract.java
package com.jermaine.awesomelibraries.view.activity.viewitem;


import com.jermaine.awesomelibraries.base.BasePresenter;
import com.jermaine.awesomelibraries.base.BaseView;

public interface ViewItemContract {
    interface ViewItemPresenter extends BasePresenter {

    }

    interface ViewItemView extends BaseView {
        String getRepoJson();

        void setId(String id);

        void setName(String name);

        void setDescription(String description);
    }
}

File: app/src/main/java/com/jermaine/awesomelibraries/uicontroller/activity/viewitem/ViewItemContract.kt
package com.jermaine.awesomelibraries.uicontroller.activity.viewitem


import com.jermaine.awesomelibraries.base.BaseView

interface ViewItemContract {
    interface ViewItemView : BaseView {
        fun setId(id: String?)

        fun setName(name: String?)

        fun setDescription(description: String?)
    }
}
